README.md

```diff
@@ -60,6 +60,25 @@ Version information can be either a single number or a semantic version string.
 }
 ```
 
+A single JSON file can provide information on multiple firmware types by combining them together into an array. When this is loaded, the firmware manifest with a type matching the one passed to the esp32FOTA constructor will be selected:
+
+```json
+[
+   {
+      "type":"esp32-fota-http",
+      "version":"0.0.2",
+      "url":"http://192.168.0.100/fota/esp32-fota-http-2.bin",
+      "check_signature":true
```

Comment:
I don't think check_signature is correct , as you set this via the  boolean `validate` to `true` in the constructor

Reply:
You are correct -- I've removed `check_signature` from each of the example json files in the readme.